example/src/main/java/com/stripe/example/activity/PaymentActivity.java

```diff
@@ -63,6 +73,16 @@ public void onClick(View view) {
             }
         });
 
+        compositeSubscription = new CompositeSubscription();
+        Button saveRxButton = (Button) findViewById(R.id.saverx);
+        compositeSubscription.add(
+                RxView.clicks(saveRxButton).subscribe(new Action1<Void>() {
```

Comment:
Not strictly necessary, but a cool handler that RxViews gives you to make things a little easier to keep track of (and ensure you disconnect properly when views are destroyed)